tensorrt_llm/_torch/pyexecutor/resource_manager.py

```
@@ -2857,7 +3166,8 @@ def update_resources(
     ):
         for _, resource_manager in self.resource_managers.items():
             if hasattr(resource_manager, "update_resources"):
-                if isinstance(resource_manager, KVCacheManager):
+                if isinstance(resource_manager,
+                               (KVCacheManager, MixedPrecisionKVCacheManager)):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)